source/developer/dev-setup-archlinux.rst

```diff
@@ -1,78 +1,3 @@
 .. _dev-setup-archlinux:
```

Comment:
Instead of having all these different developer setup pages, maybe we just have on called 'Developer Setup' that links to the dev site?

Reply:
We do at https://docs.mattermost.com/developer/dev-setup.html, but I want to keep the different set up pages, in case we link them somewhere (e.g. old forum posts)

Reply:
Is it possible to just redirect those pages straight to the dev site so that the old links still work but you don't have to click again?